app/models/package_manager/go.rb

```diff
@@ -21,32 +23,71 @@ class Go < Base
 
 
     def self.package_link(project, version = nil)
-      "http://go-search.org/view?id=#{project.name}"
+      "https://pkg.go.dev/#{project.name}#{"@#{version}" if version}"
     end
 
     def self.documentation_url(name, version = nil)
-      "http://godoc.org/#{name}"
+      "https://pkg.go.dev/#{name}#{"@#{version}" if version}?tab=doc"
     end
 
     def self.install_instructions(project, version = nil)
       "go get #{project.name}"
     end
 
     def self.project_names
-      get("http://go-search.org/api?action=packages")
+      # Currently the index only shows the last <=2000 modules from the date given. (https://proxy.golang.org/)
```

Comment:
About how many a day are there usually?

Reply:
Uh oh, just realized this is actually package-specific (not module-specific), and it's version-specific too. So this list is gonna change a lot. Left a note about that in the Go issue above tho.